src/components/radio-group/radio-group.component.ts

```diff
@@ -129,11 +129,11 @@ export default class SlRadioGroup extends ShoelaceElement implements ShoelaceFor
   }
 
   private handleRadioClick(event: MouseEvent) {
-    const target = (event.target as HTMLElement).closest<SlRadio | SlRadioButton>('sl-radio, sl-radio-button')!;
+    const target = event.target.closest<SlRadio | SlRadioButton>('sl-radio, sl-radio-button')!;
```

Comment:
Seems like removing the `as HTMLElement` here broke the TypeScript check. Mind adding it back in and see if CI passes?